doc/classes/DisplayServer.xml

```diff
@@ -849,6 +849,37 @@
 				[b]Note:[/b] This method is implemented only on macOS.
 			</description>
 		</method>
+		<method name="ios_get_gesture_state" qualifiers="const">
+			<return type="int" enum="DisplayServer.IOSUIGestureState" />
+			<description>
+				Returns the state of the UI gestures currently in effect.
+				[b]Note:[/b] This method is only implemented on iOS.
+			</description>
+		</method>
+		<method name="ios_get_status_bar_appearance" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns if the status bar is currently hidden. This can be used to modify `display/window/ios/hide_status_bar` at runtime.
```

Comment:
```suggestion
				Returns if the status bar is currently hidden. This can be used to modify [code]display/window/ios/hide_status_bar[/code] at runtime.
```